charon/utils/archive.py

```diff
@@ -81,12 +87,33 @@ def _get_tgz_name(path: str):
     return ""
 
 
-def __parse_npm_package_version_paths(path: str) -> list:
+def _del_none(d):
```

Comment:
I remember there is a same func in npm.py, if so, please remove it and reuse this one.